### ISSUE TITLE
Error when using ThenInclude on 1.1.0-rtm-10000

### ISSUE THREAD

**Issue by caleblloyd** [PomeloFoundation/Pomelo.EntityFrameworkCore.MySql#118]
Hit this error in one of my Applications using `ThenInclude` in `1.1.0-rtm-10000`.  Need to debug this and fix before shipping 1.1.0

```
      ErrorMessage: Value cannot be null.
      Parameter name: identifier
      InnerException: 
      StackTrace:    at Microsoft.EntityFrameworkCore.Utilities.Check.NotEmpty(String value, String parameterName)
         at Microsoft.EntityFrameworkCore.Storage.Internal.MySqlSqlGenerationHelper.DelimitIdentifier(String identifier)
         at Microsoft.EntityFrameworkCore.Query.Sql.DefaultQuerySqlGenerator.VisitSelect(SelectExpression selectExpression)
         at Remotion.Linq.Parsing.ThrowingExpressionVisitor.Visit(Expression expression)
         at Microsoft.EntityFrameworkCore.Query.Sql.DefaultQuerySqlGenerator.VisitLeftOuterJoin(LeftOuterJoinExpression leftOuterJoinExpression)
         at Remotion.Linq.Parsing.ThrowingExpressionVisitor.Visit(Expression expression)
         at Microsoft.EntityFrameworkCore.Query.Sql.DefaultQuerySqlGenerator.VisitJoin[T](IReadOnlyList`1 items, Action`1 itemAction, Action`1 joinAction)
         at Microsoft.EntityFrameworkCore.Query.Sql.DefaultQuerySqlGenerator.VisitSelect(SelectExpression selectExpression)
         at Remotion.Linq.Parsing.ThrowingExpressionVisitor.Visit(Expression expression)
         at Microsoft.EntityFrameworkCore.Query.Sql.DefaultQuerySqlGenerator.GenerateSql(IReadOnlyDictionary`2 parameterValues)
         at Microsoft.EntityFrameworkCore.Query.Internal.ShaperCommandContext.GetRelationalCommand(IReadOnlyDictionary`2 parameters)
         at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable.AsyncEnumerator.<BufferlessMoveNext>d__9.MoveNext()
```

**Comment by matthewjcooper** [comment]
I have the same issue with this happening on a one to one related entity. It worked fine before upgrading to 1.1. As a work around, I had to break out the loading of the elements in a separate query and use EF fixup to related the objects.

**Comment by Kagamine** [comment]
@matthewjcooper We have released 1.1.0-rtm-10001, Could you have a try?

**Comment by matthewjcooper** [comment]
Yup, working now. Thanks!
